app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -154,6 +166,16 @@ def self.replace_context_needs(path)
         end
       end
 
+      def self.replace_context_github(var_name)  
+        translated_var = @github_bk_context_mapping[var_name]
+        case translated_var
+        when String
+          "$#{translated_var}"
+        when nil
+          raise NoMethodError
```

Comment:
I believe this could be changed to not use the `case` statement:
```suggestion
        raise NoMethodError unless @github_bk_context_mapping.include?(var_name)
        
        "$$#{@github_bk_context_mapping[var_name]}"
```

Reply:
This idea is better, switched to this